packages/core/state/interaction/logics.ts

```diff
@@ -70,20 +70,30 @@ const initializeApp = createLogic({
         const isOnWeb = interactionSelectors.isOnWeb(deps.getState());
         const fileService = interactionSelectors.getHttpFileService(deps.getState());
 
-        // Redimentary check to see if the user is an AICS Employee by
+        // Rudimentary check to see if the user is an AICS employee by
         // checking if the AICS network is accessible
         const isAicsEmployee = await fileService.isNetworkAccessible();
 
-        // If there are query args representing a query we can extract that
-        // into the query to render (ex. when refreshing a page)
+        // If the url has search params representing a query, we can extract those
+        // into a query object to render (ex. when refreshing a page)
         if (isOnWeb && window.location.search) {
             try {
-                dispatch(
-                    selection.actions.addQuery({
-                        name: DEFAULT_QUERY_NAME,
-                        parts: SearchParams.decode(window.location.search),
-                    })
-                );
+                // If the exact query already exists in state on initialization,
+                // we shouldn't add it again, just refresh it
+                const equivalentExists = queries.some((query) => {
+                    return `?${SearchParams.encode(query?.parts)}` === window.location.search;
+                });
+                // Refresh existing query
+                if (queries.length && equivalentExists) {
```

Comment:
does equivalentExists imply queries.length here?

Reply:
ah yes, I think this was a legacy from debugging. fixed in [be28c24](https://github.com/AllenInstitute/biofile-finder/pull/569/commits/be28c24513166bd9e1749a56f180ee0a0b04d217)